GirafAPI/Endpoints/OrganizationEndpoints.cs

```diff
@@ -26,14 +26,15 @@ public static RouteGroupBuilder MapOrganizationEndpoints(this WebApplication app
                     }
 
                     await dbContext.Entry(user)
-                        .Collection(u => u.Organizations).LoadAsync();
+                        .Collection(u => u.Organizations!).LoadAsync();
```

Comment:
Same thing

Reply:
It was coded so that Organizations can be null. 
if (user.Organizations is null)
                    {
                        return Results.NotFound();
                    }
And this if statement was already in place, i just took care of the warning

Reply:
Fair, I think you guys should aim to "fix" the warning instead of telling the compiler that it is wrong by using !. This could be achieved by making the field required, which would signal to other developers that this is the case and make the error go away. Hope it makes sense :)